### PR TITLE
SISRP-19916, verify_sis_api_endpoints.sh is ready for GL5.5 UAT cutover check (11AM on Mon May 23)

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.berkeley.edu/browse/SISRP-19916

@davidlimcheng do we have valid replacement for UC_FPP_ENROLLMENT_API_GET?

### CODE REVIEW

**Comment by johncrossman** [comment]
@davidlimcheng this is the script that you or @dkawase will run on Monday.  Feel free to merge this PR if it makes sense.

File: app/models/campus_solutions/fpp_enrollment.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def build_feed(response)
       response.parsed_response
     end
 
-    #TODO: Replace with actual API endpoint when ready.
+    # TODO: Replace with actual API endpoint and update script/sis/verify_sis_api_endpoints.sh accordingly.
     def url
       "#{@settings.base_url}/UC_FPP_ENROLLMENT_API.v1/UC_FPP_ENROLLMENT_API_GET"
     end
```
Comment (davidlimcheng) [inline app/models/campus_solutions/fpp_enrollment.rb]: Just opened a PR for this, the endpoint will be:  `#{@settings.base_url}/UC_SF_FPP_LINKS_GET.v1/Get`

File: script/sis/verify_sis_api_endpoints.sh [hunk 2]
```diff
@@ -268,6 +268,7 @@ verify_hub "/${CAMPUS_SOLUTIONS_ID}/academic-status" \
   "/${CAMPUS_SOLUTIONS_ID}/all" \
   "/${CAMPUS_SOLUTIONS_ID}/contacts" \
   "/${CAMPUS_SOLUTIONS_ID}/demographic" \
+  "/${CAMPUS_SOLUTIONS_ID}/registrations?term-id=2168" \
```
Comment (davidlimcheng) [inline script/sis/verify_sis_api_endpoints.sh]: There's also a PR that removes the `term-id` param from the `registrations` API.  It has yet to be pulled but I think it's safe to move forward with removing `term-id`.  cc @raydavis

**Comment by davidlimcheng** [comment]
Thanks @johncrossman , apologies for the slow response.  Left some comments - and I do not have merge powers!

**Pull request closed without merging**